target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -45,6 +45,13 @@
 			pinctrl-0 = <&mdio0_pins &rgmii2_pins>;
 			pinctrl-names = "default";
 
```

Comment:
It's used this way in chromium and Qualcomm sdk repositories
https://source.codeaurora.org/quic/qsdk/system/openwrt/tree/arch/arm/boot/dts?h=korg/linux-3.4.y/release/arugula_bb_cs
https://chromium.googlesource.com/chromiumos/third_party/kernel/+/chromeos-3.14/arch/arm/boot/dts
But beware, not everything can be used from there

R7500v2 seems ok